baseapp/info.go

```diff
@@ -0,0 +1,187 @@
+package baseapp
+
+import (
+	"time"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+
+	"cosmossdk.io/core/comet"
+)
+
+type reqBeginBlockInfo struct {
+	abci.RequestBeginBlock
```

Comment:
this should be removed, this just makes it so we have to remove this type in a follow up pr for comet 0.38 integration